File: .claude/commands/unit-test-writer/tests/fixtures/go/calculator_test.go
// Intentionally incomplete — covers only Add and Subtract.
// unit-test-writer should detect Multiply, Divide, and Power are untested
// and generate tests to bring coverage to ≥90%.

package calculator

import "testing"

func TestAdd(t *testing.T) {
	result := Add(2, 3)
	if result != 5 {
		t.Errorf("Add(2, 3) = %v, want 5", result)
	}
}

func TestSubtract(t *testing.T) {
	result := Subtract(10, 4)
	if result != 6 {
		t.Errorf("Subtract(10, 4) = %v, want 6", result)
	}
}

File: .claude/commands/unit-test-writer/tests/fixtures/javascript/src/calculator.test.js
// Intentionally incomplete — covers only add and subtract.
// unit-test-writer should detect multiply, divide, and power are untested
// and generate tests to bring coverage to ≥90%.

const { add, subtract } = require('./calculator');

describe('add', () => {
  it('returns the sum of two positive numbers', () => {
    expect(add(2, 3)).toBe(5);
  });

  it('returns the correct value when adding zero', () => {
    expect(add(0, 5)).toBe(5);
  });
});

describe('subtract', () => {
  it('returns the difference of two numbers', () => {
    expect(subtract(10, 4)).toBe(6);
  });

  it('returns a negative result when minuend is smaller', () => {
    expect(subtract(3, 7)).toBe(-4);
  });
});
